src/main/java/nl/novi/techItEasy/models/Television.java

```diff
@@ -30,13 +30,18 @@ public class Television {
     private Integer sold;
 
     // Cascade komt uit EdHub H14, volgens mij zorgt dat ervoor dat je bij een delete ook de gerelateerde info delete
-    // In huiswerkles werd private voor de variabele gezet. In H14 niks (public?)
+    // In huiswerkles werd private voor de variabele gezet. In H14 niks (package private)
     @OneToOne(cascade = CascadeType.ALL)
     private RemoteController remote;
 
+    // Dit moet de owner kant van de relatie zijn (met foreign key in de database): @ManyToOne
+//    @ManyToOne(fetch = FetchType.EAGER)
+//    @JoinColumn(name = "ci_module_id")
+//    private CIModule ciModule;
     @OneToMany(mappedBy = "tv")
     private List<CIModule> ciModules;
 
+    // LazyCollection, JsonIgnore toevoegen. En het is een collectie, geen lijst?
```

Comment:
De CrudRepository geeft een Collection terug voor de findAll() methode. De JpaRepository geeft een List terug voor de findAll() methode. Collection is een superklasse van List, dus dat kan prima, alleen heeft het minder functionaliteit.

Reply:
Correctie: Crudrepository.findAll() returned een Iterable, geen Collection. Een Iterable heeft nog minder functionaliteit dan een Collection